code/game/objects/items/devices/flash.dm

```diff
@@ -86,8 +86,6 @@
 
 		if(safe <= 0)
 			Subject.Knockdown(10)
-			Subject.flash_eyes(visual = 1, affect_silicon = 1)
```

Comment:
I think this was literally flashing people twice or something, @DrCelt get in here